core/driver/sources/src/main/resources/reference.conf

```diff
@@ -0,0 +1,81 @@
+# Default configuration for the org.virtuslab.ideprobe.config.IdeProbeConfig
+probe {
+
+  # intellij is for org.virtuslab.ideprobe.config.IntellijConfig - default values below
+  intellij {
+    plugins = []
+    version {
+      build = "212.5080.55"
+      release = "2021.2.1"
+    }
+    # path = "" // use this config with proper path string if you want to use non-default IntellijConfig
+  }
+
+  # workspace is for Option[org.virtuslab.ideprobe.config.WorkspaceConfig] - default None - comments added as a reference
+  # workspace {
+  #   path { // for org.virtuslab.ideprobe.dependencies.Resource
+  #     uri {
+  #       str = ""
+  #     }
+  #     path {} // for java.nio.file.Path interface - choose proper implementation and add proper parameter names
+  #     cause {
+  #       s = ""
+  #     }
+  #   }
+  #   existing {} // for java.nio.file.Path interface - choose proper implementation and add proper parameter names
+  #   branch = ""
+  #   tag = ""
+  #   commit = ""
+  # }
+
+  # resolvers id for org.virtuslab.ideprobe.config.DependenciesConfig.Resolvers - default values below
+  resolvers {
+    intellij {
+      repositories = []
+    }
+    plugins { }
+    jbr {
+      repositories = []
+    }
+    retries = 0
+  }
+
+  # driver is for org.virtuslab.ideprobe.config.DriverConfig - default values below
+  driver {
+    launch {
+      command = []
+      timeout = "30.seconds"
+    }
+    check {
+      errors {
+        enabled = false
+        includeMessages = [".*"]
+        excludeMessages = []
+      }
+      freezes {
+        enabled = false
+      }
+    }
+    xvfb {
+      screen {
+        width = 1920
+        height = 1080
+        depth = 24
+      }
+    }
+    headless = false
+    vmOptions = []
+    env = {}
+  }
+
+  # paths is for org.virtuslab.ideprobe.config.PathsConfig - default values below
+  paths {
+    # all parameters below are Options and are None by default - hence commented out
+    # base {} // for java.nio.file.Path interface - choose proper implementation and add proper parameter names
```

Comment:
See https://github.com/VirtusLab/git-machete-intellij-plugin/blob/master/src/uiTest/resources/ideprobe.conf#L2 ... it looks that `pureconfig` accepts just strings where a Path is expected

Reply:
You're right. So let me just simplify these comments and suggest adding proper string instead.